lampo-common/src/model/close_channel.rs

```diff
@@ -0,0 +1,53 @@
+pub mod request {
+    use std::num::ParseIntError;
+    use std::str::FromStr;
+
+    use bitcoin::secp256k1::PublicKey;
+    use serde::{Deserialize, Serialize};
+
+    use crate::error;
+    use crate::types::*;
+
+    #[derive(Clone, Serialize, Deserialize)]
+    pub struct CloseChannel {
+        pub counterpart_node_id: String,
+        // Hex of the channel
+        pub channel_id: String,
```

Comment:
```suggestion
        pub channel_id: Option<String>,
```

This is optional, if there is a single channel with the peer, you can grep the channel id

Reply:
Can you please explain a bit more as why do we need this?

Reply:
The channel is is optional, because if you have a single channel with a peer you know already what channel you can close, see the close of core lightning

Reply:
Okay I have implemented this in fdc3963e68e242d1099a57d27a1dd0754e88a071